src/main/java/org/recap/controller/BulkRequestController.java

```diff
@@ -138,16 +131,16 @@ public ModelAndView searchRequestByPatronBarcode(String patronBarcodeInRequest,M
     }
 
     @PostMapping("/bulkRequest/loadCreateRequestForSamePatron")
-    public ModelAndView loadCreateRequestForSamePatron(@Valid @ModelAttribute("bulkRequestForm") BulkRequestForm bulkRequestForm,Model model) {
+    public ModelAndView loadCreateRequestForSamePatron(@Valid @ModelAttribute("bulkRequestForm") BulkRequestForm bulkRequestForm, Model model) {
```

Comment:
@DinakarCiber - I see you have the request mapping bulkRequest at the class level, then why need to bulkRequest again in the end point. This will make the endpoint bulkRequest/bulkRequest/....